packages/schematics/angular/application/other-files/package.json.template

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "<%= utils.dasherize(name) %>",
+  "private": true,
+  "description": "This is a fake package.json file which is not used by package managers. It is used to tell the bundlers whether the application is free from side-effect code.",
+  "sideEffects": <%= !strict %>
```

Comment:
Technically only need the `sideEffects` field here.  I think the description is useful but I'm undecided about the others.

Reply:
I dont mind removing the others.

Reply:
@IgorMinar thoughts?

Reply:
I think we should keep all otherwise nobody will understand what this package.json is here for and people will remove it because they'll think that it's junk.